soh/src/overlays/actors/ovl_En_Mb/z_en_mb.c

```diff
@@ -704,6 +711,14 @@ void EnMb_SpearEndChargeQuick(EnMb* this, PlayState* play) {
 }
 
 void EnMb_ClubWaitAfterAttack(EnMb* this, PlayState* play) {
+
+    // Rotate Club Moblin towards player in Enemy Randomizer because they're borderline
+    // useless otherwise in most scenarios.
```

Comment:
```suggestion
    // Rotate Club Moblin towards player in Enemy Randomizer because they're
    // borderline useless otherwise in most scenarios.
```